hail/src/main/scala/is/hail/expr/ir/TableIR.scala

```diff
@@ -1099,6 +1105,7 @@ case class MatrixRowsTable(child: MatrixIR) extends TableIR {
   }
 
   val typ: TableType = child.typ.rowsTableType
+  def rvdType: RVDType = throw new NotImplementedError()
```

Comment:
Can you get away with this?  This screams (1) latent bug or (2) rvdType is actually unused.

Reply:
yeah, I'll see if I can peel it off table too

Reply:
yes, rvdType gone from table too!

Reply:
Woot!